packages/qwik-city/src/middleware/request-handler/types.ts

```diff
@@ -201,6 +202,12 @@ export interface RequestEventCommon<PLATFORM = QwikCityPlatform>
    */
   readonly redirect: (statusCode: RedirectCode, url: string) => RedirectMessage;
 
+  /**
+   * URL to rewrite to. When called, the response will immediately end with the correct rewrite url.
```

Comment:
The response should stay active, no?

Reply:
Yes, I initially taken the phrasing from redirect - I'm still figuring things out

Reply:
Fixed